app/src/main/java/com/example/whereto/models/YelpFusion.java

```diff
@@ -0,0 +1,32 @@
+package com.example.whereto.models;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Properties;
+
+public class YelpFusion {
```

Comment:
Could you help me understand the naming, why YelpFusion?

Reply:
Version 3 of the Yelp API is named Yelp Fusion!